elements/lisk-validator/src/formats.ts

```diff
@@ -125,3 +127,41 @@ const camelCaseRegex = /^[a-z]+((\d)|([A-Z0-9][a-zA-Z0-9]+))*([a-z0-9A-Z])?$/;
 
 export const camelCase = (data: string): boolean =>
 	camelCaseRegex.exec(data) !== null;
+
+export const version = (data: string): boolean =>
+	/^([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})(-(alpha|beta|rc)\.[0-9]{1,3}(\.[0-9]{1,3})?)?$/.test(
+		data,
+	);
+
+export const protocolVersion = (data: string): boolean =>
+	/^(\d|[1-9]\d{1,2})\.(\d|[1-9]\d{1,2})$/.test(data);
+
+export const path = (data: string): boolean => {
+	const pathRegExp = new RegExp('^(.?)(/[^/]+)+$');
+	return pathRegExp.test(data);
+};
+
+export const encryptedPassphrase = (data: string): boolean => {
+	// Explanation of regex structure:
+	// - 1 or more 'key=value' pairs delimited with '&'
+	// Examples:
+	// - cipherText=abcd1234
+	// - cipherText=abcd1234&iterations=10000&iv=ef012345
+	// NOTE: Maximum lengths chosen here are arbitrary
+	const keyRegExp = /[a-zA-Z0-9]{2,15}/;
+	const valueRegExp = /[a-f0-9]{1,256}/;
+	const keyValueRegExp = new RegExp(
+		`${keyRegExp.source}=${valueRegExp.source}`,
+	);
+	const encryptedPassphraseRegExp = new RegExp(
+		`^(${keyValueRegExp.source})(?:&(${keyValueRegExp.source})){0,10}$`,
+	);
+	return encryptedPassphraseRegExp.test(data);
+};
```

Comment:
This does not match with required dataset. 

<img width="877" alt="image" src="https://user-images.githubusercontent.com/112468/84251684-fb769a80-ab0d-11ea-9155-a549d09f643c.png">


Try this one

```
[a-z0-9A-z]*=[^&]+&*
```
<img width="877" alt="image" src="https://user-images.githubusercontent.com/112468/84252484-ebab8600-ab0e-11ea-987b-7b15bc11663d.png">